docs/en/multi_modal/internvl.md

```diff
@@ -1,22 +1,43 @@
-# InternVL2
+# InternVL
 
-## Introduction
+LMDeploy supports the following InternVL series of models, which are detailed in the table below:
 
-InternVL is an open source vision-language base model that expands the Vision Transformer (ViT) to 600 million parameters and aligns with the Large Language Model (LLM). It is the largest open-source vision/vision-language foundation model (14B) to date, achieving 32 state-of-the-art performance on a wide range of tasks such as visual perception, cross-modal retrieval, multimodal dialogue, etc. LMDeploy supports InternVL series of models. The following uses InternVL2-8B as an example to demonstrate its usage.
+|    Model    |  Size   | Supported Inference Engine |
+| :---------: | :-----: | :------------------------: |
+|  InternVL   | 13B-19B |         TurboMind          |
+| InternVL1.5 | 2B-26B  |     TurboMind, PyTorch     |
+|  InternVL2  |   1B    |          PyTorch           |
+|  InternVL2  | 2B-76B  |     TurboMind, PyTorch     |
```

Comment:
4B is only supported by pytorch engine since it head_dim is 96
https://huggingface.co/OpenGVLab/InternVL2-4B/blob/b50544dafada6c41e80bfde2f57cc9b0140fc21c/config.json#L43

Reply:
I'll fix it in https://github.com/ColorfulDick/lmdeploy/pull/2

Reply:
> 4B is only supported by pytorch engine since it head_dim is 96 https://huggingface.co/OpenGVLab/InternVL2-4B/blob/b50544dafada6c41e80bfde2f57cc9b0140fc21c/config.json#L43



> I'll fix it in [ColorfulDick#2](https://github.com/ColorfulDick/lmdeploy/pull/2)

i have merge the pr